### PR TITLE
General execute command

### PR DESCRIPTION
**Pull request by DvirDukhan**
minor code fixes + binary commands now include general command with no args.
an update to #2007

### CODE REVIEW

File: src/main/java/redis/clients/jedis/commands/BinaryJedisCommands.java [hunk 1]
```diff
@@ -358,4 +358,6 @@
   List<byte[]> xclaim(byte[] key, byte[] groupname, byte[] consumername, long minIdleTime, long newIdleTime, int retries, boolean force, byte[][] ids);
 
   Object sendCommand(ProtocolCommand cmd, byte[]... args);
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/commands/BinaryJedisCommands.java]: I missed this earlier. I think, we shouldn't include these here. These interfaces imitates actual Redis commands. But `sendCommand` is not a Redis command. I.e. there isn't any `SEND COMMAND` command. We may have to come up with some other solutions for this.

Comment (DvirDukhan) [reply src/main/java/redis/clients/jedis/commands/BinaryJedisCommands.java]: @sazzad16 a proposal: I'll remove the sendCommand entries from all affected interfaces. The definition and implementation will be only in the classes themselves.

Comment (sazzad16) [reply src/main/java/redis/clients/jedis/commands/BinaryJedisCommands.java]: @DvirDukhan I'm okay with that, specially in beginning. We may need a new interface for this.

@gkorland thoughts?

Comment (DvirDukhan) [reply src/main/java/redis/clients/jedis/commands/BinaryJedisCommands.java]: @sazzad16 I will open an issue of the new interface(s) required, so we can discuss it there. Meanwhile, I will remove the additional commands from the interface

**Pull request merged**